Add `osu! originals - osu!taiko World Cup 2023` news article (#9282)
ppy/osu-wiki: news/2023/2023-05-04-osu-originals-osu-taiko-world-cup-2023.md

---
layout: post
title: "osu! originals - osu!taiko World Cup 2023"
date: 2023-05-04 11:00:00 +0000
---

> Japan won the [osu!taiko World Cup](/wiki/Tournaments/TWC) for the 10th year in a row, but this is the first TWC with **5 brand-new songs** created specifically for the tournament!

Reviewer: ```suggestion
Japan may have won the [osu!taiko World Cup](/wiki/Tournaments/TWC) for the 10th year in a row, but this is the first TWC with **5 brand-new songs** created specifically for the tournament!
```

![](/wiki/shared/news/banners/featured-artist.jpg)

> We call those songs [**osu! originals**](/wiki/osu!_originals) because... they're original songs made for osu!. While our names lack creativity, the creativity in new songs from [2023's osu!taiko World Cup](/wiki/Tournaments/TWC/2023) make up for it.

Reviewer: ```suggestion
We call those songs [**osu! originals**](/wiki/osu!_originals) because... they're original songs made for osu!. While our names lack creativity, the creativity in the new songs from [2023's osu!taiko World Cup](/wiki/Tournaments/TWC/2023) make up for it.
```

Check out the new tunes for yourself:

<div align="center">
    <video width="95%" controls>
        <source src="LINK" type="video/mp4" preload="none">
    </video>
</div>

### Toromaru - Sleight of Hand

High-tempo jazz is [**Toromaru**](https://osu.ppy.sh/beatmaps/artists/279)'s specialty, and it perfectly suits the rhythmic complexity that osu!taiko needs.

> [Sleight of Hand](https://osu.ppy.sh/beatmapsets/1972518) mapped by [Nurend](https://osu.ppy.sh/users/9905079) slotted into the [Semifinals map pool](/wiki/Tournaments/TWC/2023#semifinals) as a no-mod pick, and led to *two* matches that were decided by less than 20,000 score difference!

Reviewer: ```suggestion
[Sleight of Hand](https://osu.ppy.sh/beatmapsets/1972518) mapped by [Nurend](https://osu.ppy.sh/users/9905079) slotted into the [Semifinals mappool](/wiki/Tournaments/TWC/2023#semifinals) as a *No Mod* pick, and led to *two* matches that were decided by less than 20,000 score difference!
```

> [Sleight of Hand](https://osu.ppy.sh/beatmapsets/1972518) mapped by [Nurend](https://osu.ppy.sh/users/9905079) slotted into the [Semifinals mappool](/wiki/Tournaments/TWC/2023#semifinals) as a *No Mod* pick, and led to *two* matches that were decided by less than 20,000 score difference!

Reviewer: the grammar is slightly off here. this can be written a couple ways:

```suggestion
[Sleight of Hand](https://osu.ppy.sh/beatmapsets/1972518) mapped by [Nurend](https://osu.ppy.sh/users/9905079) slotted into the [Semifinals mappool](/wiki/Tournaments/TWC/2023#semifinals) as a *No Mod* pick, and led to *two* matches that were decided by a score difference less than 20,000!
```

```suggestion
[Sleight of Hand](https://osu.ppy.sh/beatmapsets/1972518) mapped by [Nurend](https://osu.ppy.sh/users/9905079) slotted into the [Semifinals mappool](/wiki/Tournaments/TWC/2023#semifinals) as a *No Mod* pick, and led to *two* matches that were decided by less than 20,000 score!
```

```suggestion
[Sleight of Hand](https://osu.ppy.sh/beatmapsets/1972518) mapped by [Nurend](https://osu.ppy.sh/users/9905079) slotted into the [Semifinals mappool](/wiki/Tournaments/TWC/2023#semifinals) as a *No Mod* pick, and led to *two* matches that were decided by a less-than-20,000 score difference!
```

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

### LV.4 feat. Mawata Spica - Vortex of Sorrow

As a master of hardcore and trance, [**LV.4**](https://osu.ppy.sh/beatmaps/artists/214) ensured that anyone listening to *Vortex of Sorrow* would be immersed in endlessly pounding bass and classic synth melodies. 

> [This beatmap](https://osu.ppy.sh/beatmapsets/1980701) by [uone](https://osu.ppy.sh/users/5321719) acted as a freemod pick in the [Grand Finals map pool](/wiki/Tournaments/TWC/2023#grand-finals). If you think you can handle peak osu!taiko gameplay, give it a try.

Reviewer: ```suggestion
[This beatmap](https://osu.ppy.sh/beatmapsets/1980701) by [uone](https://osu.ppy.sh/users/5321719) acted as a *Free Mod* pick in the [Grand Finals mappool](/wiki/Tournaments/TWC/2023#grand-finals). If you think you can handle peak osu!taiko gameplay, give it a try.
```

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

> ### A? - Umami Blaze

Reviewer: shouldn't this be "Aquestion - Umami Blaze" like in the [linked beatmap](https://osu.ppy.sh/beatmapsets/1976814)?

please see https://github.com/ppy/osu-wiki/pull/9242#discussion_r1181083428 and https://discord.com/channels/188630481301012481/218677502141399041/1101886800231936130 and confirm ty

Author: ![](https://i.imgur.com/CiuD6Td.png)

Considering [**A?**](https://osu.ppy.sh/beatmaps/artists/173) is [an osu!taiko player](https://osu.ppy.sh/users/12011880), it's no surprise that they know exactly what it takes to create an engaging song for osu!taiko. 

> [Umami Blaze](https://osu.ppy.sh/beatmapsets/1976814) mapped by [youtune3](https://osu.ppy.sh/users/6442010) was part of the [Finals map pool](/wiki/Tournaments/TWC/2023#finals) as a no-mod pick. Even if speed is your specialty, this **336 BPM** map might be a bit fast for you.

Reviewer: ```suggestion
[Umami Blaze](https://osu.ppy.sh/beatmapsets/1976814) mapped by [youtune3](https://osu.ppy.sh/users/6442010) was part of the [Finals mappool](/wiki/Tournaments/TWC/2023#finals) as a *No Mod* pick. Even if speed is your specialty, this **336 BPM** map might be a bit fast for you.
```

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

> ### 7_7 feat. kanase vavzed - 7D

Reviewer 2: ```suggestion
### 7_7 feat. kasane vavzed - 7D
```

> [**7_7**](https://osu.ppy.sh/beatmaps/artists/356) was [revealed as a Featured Artist](https://osu.ppy.sh/home/news/2023-04-23-new-featured-artist-7-7) alongside the [Grand Finals map pool](/wiki/Tournaments/TWC/2023#grand-finals) showcase just a couple weeks ago, pulling yet another osu!taiko resident into our Featured Artist catalogue!

Reviewer: ```suggestion
[**7_7**](https://osu.ppy.sh/beatmaps/artists/356) was [revealed as a Featured Artist](https://osu.ppy.sh/home/news/2023-04-23-new-featured-artist-7-7) alongside the [Grand Finals mappool](/wiki/Tournaments/TWC/2023#grand-finals) showcase just a couple weeks ago, pulling yet another osu!taiko resident into our Featured Artist catalogue!
```

For pure chaos, give your best attempt to [this map](https://osu.ppy.sh/beatmapsets/1980743) created by multi-talented mapper/composer [7_7](https://osu.ppy.sh/users/7491106)!

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

### Gram - Sigrdrifa

> The **Grand Finals tiebreaker** has always been the climax of any tournament's map pool, so bringing [**DJ Genki / Gram**](https://osu.ppy.sh/beatmaps/artists/295) on board was a must.

Reviewer: ```suggestion
The **Grand Finals tiebreaker** has always been the climax of any tournament's mappool, so bringing [**DJ Genki / Gram**](https://osu.ppy.sh/beatmaps/artists/295) on board was a must.
```

> Prepare your hands for over 5 minutes of [this 8.5 star map](https://osu.ppy.sh/beatmapsets/1981053) created by [Nifty](https://osu.ppy.sh/users/4956097) and [rubies87](https://osu.ppy.sh/users/4949934). Try not to get any 100s on those last 4 notes too. :)

Reviewer: ```suggestion
Prepare your hands for over 5 minutes of [this 8.5-star map](https://osu.ppy.sh/beatmapsets/1981053) created by [Nifty](https://osu.ppy.sh/users/4956097) and [rubies87](https://osu.ppy.sh/users/4949934). Try not to get any 100s on those last 4 notes too. :)
```

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

---

Of course, if you're interested in mapping any of these new tracks, they're available as pre-timed beatmap templates on each artist's listing!

- [**Toromaru**](https://osu.ppy.sh/beatmaps/artists/279)
- [**LV.4**](https://osu.ppy.sh/beatmaps/artists/214)
- [**A? / Aquestion**](https://osu.ppy.sh/beatmaps/artists/173)
- [**7_7**](https://osu.ppy.sh/beatmaps/artists/356)
- [**DJ Genki / Gram**](https://osu.ppy.sh/beatmaps/artists/295)

> We'd also like to give [mangomizer](https://osu.ppy.sh/users/1893718) a pat on the back for coordinating the creation of these original songs with TWC's map poolers. None of this would've been possible without his help!

Reviewer: ```suggestion
We'd also like to give [mangomizer](https://osu.ppy.sh/users/1893718) a pat on the back for coordinating the creation of these original songs with TWC's mappoolers. None of this would've been possible without his help!
```

> Keen ears may have also noticed something special about this year's osu!taiko World Cup map pool: **every song was part of our Featured Artist library!** This is the first time any world cup has used exclusively licensed music, showing just how much music we've licensed over the last few years. (Thanks for that too, mango.)

Reviewer: ```suggestion
Keen ears may have also noticed something special about this year's osu!taiko World Cup mappool: **every song was part of our Featured Artist library!** This is the first time any world cup has used exclusively licensed music, showing just how much music we've licensed over the last few years. (Thanks for that too, mango.)
```

> That's all for now! A new Featured Artist arrives this weekend, so stay tuned for that.

Reviewer: ```suggestion
That's all for now! A new Featured Artist will be arriving this weekend, so stay tuned for that.
```
